languages/en.ini

```diff
@@ -1199,9 +1199,14 @@ seconds_abbrev = "s"
 see all = "see all"
 See also = "See also"
 see_all_ellipsis = "see all…"
+select_item_checked_out_renew = "Select item for renewing"
+select_item_hold_cancel = "Select item for canceling holds or recall"
+select_item_ill_request_cancel = "Select item for canceling interlibrary loan requests"
+select_item_storage_retrieval_request_cancel = "Select item for canceling storage retrieval requests"
 Select this record = "Select this record"
 Select your carrier = "Select your carrier"
-select_page = "Select Page"
+select_page = "Select all entries of the page"
+select_page_cart = "Select all entries of the clipboard"
```

Comment:
I would change these to "on the page" (rather than "of the page") and "in the book bag" (rather than "of the clipboard") for consistency/clearer English.